SUMMARY.md

```diff
@@ -14,6 +14,13 @@
   * [Authors and Reviewers](infrastructure/authors.md)
 * [What, When and How to Measure](measuring/introduction.md)
   * [Goals using Metrics](measuring/goals.md)
+    * [Use Goals, Questions, and Metrics](measuring/use\_gqm.md)
+      * [Goals](use\_gqm/goals/README.md)
```

Comment:
I wonder if there's a way to auto-generate this part

Reply:
Yes, and we will!